src/renderers/dom/client/utils/getEventTarget.js

```diff
@@ -20,6 +20,12 @@
  */
 function getEventTarget(nativeEvent) {
   var target = nativeEvent.target || nativeEvent.srcElement || window;
+
+  // Normalize SVG <use> element events #4963
+  if (target.correspondingUseElement && !target.nodeType) {
```

Comment:
It seems to me like the `!target.nodeType` doesn't to anything really. If the target has a `correspondingUseElement` property, the target must be an instance of  `SVGElementInstance` and lives in the shadow DOM. Such elements do not have a `nodeType` property, and therefore it will always be falsy. Perhaps the check is present for some weird browser bugs? @edmellum 

https://www.w3.org/TR/SVG/single-page.html#struct-__svg__SVGElementInstance__correspondingUseElement